profiles/schema.py

```diff
@@ -0,0 +1,71 @@
+import graphene
+from graphene import relay
+from graphene_django.filter import DjangoFilterConnectionField
+from graphene_django.types import DjangoObjectType
+
+from .models import Profile, DivisionOfInterest
+from thesaurus.models import Concept
+from munigeo.models import AdministrativeDivision
+
+
+class ConceptType(DjangoObjectType):
+    class Meta:
+        model = Concept
+
+
+class AdministrativeDivisionType(DjangoObjectType):
+    class Meta:
+        model = AdministrativeDivision
+
+
+class DivisionOfInterestType(DjangoObjectType):
+    class Meta:
+        model = DivisionOfInterest
+
+
+class ProfileType(DjangoObjectType):
+    class Meta:
+        model = Profile
+
+    concepts_of_interest = graphene.List(ConceptType)
+    divisions_of_interest = graphene.List(AdministrativeDivisionType)
+
+
+class ProfileInput(graphene.InputObjectType):
+    user_id = graphene.ID()
+    nickname = graphene.String(required=True)
+    image = graphene.String()
+    email = graphene.String()
+    phone = graphene.String()
+    language = graphene.String()
+    contact_method = graphene.String()
+
+
+class CreateProfile(graphene.Mutation):
+    class Arguments:
+        profile = ProfileInput(required=True)
+
+    ok = graphene.Boolean()
+    profile = graphene.Field(ProfileType)
+
+    def mutate(self, info, **kwargs):
+        profile = Profile.objects.create(**kwargs)
+
+        ok = True
+        return CreateProfile(profile=profile, ok=ok)
+
+
+class Query(graphene.ObjectType):
+    profile = relay.Node.Field(ProfileType)
+    profiles = graphene.List(ProfileType)
```

Comment:
all the profile endpoints right now are blocked for [non-authenticated users](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/api.py#L92). GETing and POSTing requires authentication and we should keep that logic in GraphQL. 
here is a package, that we are gonna need for that: https://github.com/flavors/django-graphql-jwt

Reply:
then you'll just add [`@login_required` decorator](https://django-graphql-jwt.domake.io/en/stable/decorators.html#login-required) where needed

Reply:
we'll need to connect GraphQLto helusers, I guess, to make [this middleware](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/open_city_profile/settings.py#L168) to work

Reply:
Agreed that authentication status of users should be taken into account. At the moment non-admins can only view and mutate their own profiles, but in the future we will most probably have more permission classes (e.g. non-superuser city officials should be able to create new profiles).